packages/backend/src/index.ts

```diff
@@ -19,7 +19,9 @@
 import {createServer} from '@/server';
 
 // Set up the dotenv variables when running in development mode.
-require('dotenv')?.config();
+if (process.env.NODE_ENV === 'development') {
+  require('dotenv').config();
```

Comment:
why require dotenv only for development? usually do you have envs for both environments

just curious about it

Reply:
when i deploy to production i don't touch de filesystem, and set by the hosting service like heroku, vercel, etc